README.md

```diff
@@ -1,9 +1,6 @@
 # Composer-enabled Upstream template
-## Experimental
 
 This is Pantheon's recommended starting point for forking new upstreams that work with the Platform's integrated
 Composer build process.
 
-Because it is under active development, you should not create permanent/production sites using this repository
-yet. We make no guarantees of backwards compatibility. Merging new updates from this repository may break existing
-sites.
+For more information and detailed installation guides, please visit the Integrated Composer Pantheon documentation: https://pantheon.io/docs/integrated-composer
```

Comment:
Docs article not published yet, see https://github.com/pantheon-systems/documentation/pull/5663